webpack/src/reducers.js

```diff
@@ -0,0 +1,7 @@
+import { combineReducers } from "redux";
+
+const reducers = {
+  foremanDiscovery: combineReducers({}),
```

Comment:
Currently the console show the following error:
```console
Store does not have a valid reducer. Make sure the argument passed to combineReducers is an object whose values are reducers.
```

For now because you are not passing any reducer it would be OK to return an empty object:
```suggestion
  foremanDiscovery: {},
```